app/controllers/api/v1/get_an_identity/webhook_messages_controller.rb

```diff
@@ -1,7 +1,7 @@
 module API
   module V1
     module GetAnIdentity
-      class WebhookMessagesController < BaseController
+      class WebhookMessagesController < ApplicationController
```

Comment:
not sure how this happened 😅

Reply:
I changed this to `ApplicationController` just incase it messed up the existing behaviour.
Every controller inheriting `BaseController` will be authenticated only.

Reply:
Move `defaults: { format: "json" }` to only migration API endpoints